packages/admin/admin-date-time/src/DatePickerNavigation.tsx

```diff
@@ -107,7 +107,7 @@ export const DatePickerNavigation = (inProps: DatePickerNavigationProps) => {
     );
 };
 
-export type DatePickerNavigationClassKey =
+type DatePickerNavigationClassKey =
```

Comment:
The class key should be added to src/index.ts

Reply:
exported